### ISSUE TITLE
Very old pngjs dependency 0.4.0

### ISSUE THREAD

**Issue by samypr100** [joshmarinacci/node-pureimage#30]
Mostly informational, but I'm getting errors from yarn and npm related to pngjs:

```bash
pngjs@0.4.0: The engine "node" is incompatible with this module. Expected version "0.8.x".
```

**Comment by cristianmartinez** [comment]
Any update?

**Comment by robertmain** [comment]
Looks like @lightsofapollo put a PR together to address this...

**Comment by robertmain** [comment]
@samypr100  @cristianmartinez looks like it's been fixed now :)

**Comment by robertmain** [comment]
Just discovered a potential issue with this....

I'm re-writing the unit tests for the library.  My unit tests are now failing after this dependency update. 

Currently investigating the issue....more info to follow.

Tagging @joshmarinacci for visibility.

**Comment by cristianmartinez** [comment]
@robertmain something I can help you with?

**Comment by robertmain** [comment]
Perhaps. Currently looking into it. Take a look at the `fix-unit`tests` branch on my fork. Either way, if the tests are falling it's possible something has changed in the way pngjs behaves. Figured you might want a heads up if you're using it in a production system...

**Comment by robertmain** [comment]
Okay, panic over - it looks like it was because I was using Jest wrong causing it to show a different test failing than was actually the case....and the PNG library changed how they deal with errors. I've tweaked [the test around the error condition](https://github.com/joshmarinacci/node-pureimage/pull/49/files#diff-85c7219204ef2a7cb734cbc040c6e396R41) and things are fine now

**Comment by robertmain** [comment]
Looks like this is fixed now. Can the issue be closed?

**Comment by guigrpa** [comment]
Apparently it's not closed:

```bash
$ yarn add --dev pureimage
yarn add v1.3.2
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
error pngjs@0.4.0: The engine "node" is incompatible with this module. Expected version "0.8.x".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

**Comment by robertmain** [comment]
Oh, you're using yarn? I guess that explains why I haven't run into this...


It looks like yarn has [an outdated copy of `package.json`](https://cdn.jsdelivr.net/npm/pureimage/package.json). If you compare it to [the version on GitHub](https://github.com/joshmarinacci/node-pureimage/blob/master/package.json) you can see that there are quite a few changes. Either `yarn` hasn't picked that up, or it hasn't been told to re-scan the package(I'm not entirely sure how `yarn` works.

**Comment by robertmain** [comment]
@guigrpa Can you confirm that this is fixed for you?

**Comment by guigrpa** [comment]
Perfect! Tested with yarn: no problem.

**Comment by samypr100** [comment]
Fixed when 0.1.4+ is specified in yarn! Thanks! Feel free to close 👍
